lib/language-inference-engine.js

```
@@ -185,41 +185,47 @@ var ENGINE_PROFILES = {
         "type": "llm",
         "availableModels": [
+            "gemini-2.0-flash",
             "gemini-1.5-flash"
         ],
         "headers": {
             "Content-Type": "application/json"
         },
         "url": "https://generativelanguage.googleapis.com/v1beta/models/{model}:generateContent?key={apikey}",
-        "warp": function(model, message, temperature) {
+        "wrap": function(model, message, temperature) {
+            const BIAS_MESSAGE = BIAS_MESSAGE
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)